mavros_extras/src/plugins/mocap_fake_gps.cpp

```diff
@@ -0,0 +1,259 @@
+/**
+ * @brief MocapPoseEstimate plugin
+ * @file mocap_pose_estimate.cpp
+ * @author Tony Baltovski <tony.baltovski@gmail.com>
+ * @author Vladimir Ermakov <vooon341@gmail.com>
+ *
+ * @addtogroup plugin
+ * @{
+ */
+/*
+ * Copyright 2014,2015 Vladimir Ermakov, Tony Baltovski.
+ *
+ * This file is part of the mavros package and subject to the license terms
+ * in the top-level LICENSE file of the mavros repository.
+ * https://github.com/mavlink/mavros/tree/master/LICENSE.md
+ */
+
+#include <mavros/mavros_plugin.h>
+#include <pluginlib/class_list_macros.h>
+#include <eigen_conversions/eigen_msg.h>
+
+#include <geometry_msgs/PoseStamped.h>
+#include <geometry_msgs/TransformStamped.h>
+#include <vicon_bridge/TfDistortInfo.h>
+
+
+namespace mavplugin {
+/**
+ * @brief MocapFakeGPS plugin
+ *
+ * Sends fake GPS from motion capture data to FCU.
+ */
+
+
+class MocapFakeGPSPlugin : public MavRosPlugin
+{
+public:
+	MocapFakeGPSPlugin() :
+		mp_nh("~fake_gps"),
+		uas(nullptr)
+	{ };
+
+	void initialize(UAS &uas_)
+	{
+
+		uas = &uas_;
+
+		mocap_tf_d_sub = mp_nh.subscribe("/tf_distort/out", 1, &MocapFakeGPSPlugin::mocap_tf_d_cb, this);
+
+		mocap_tf_params_sub = mp_nh.subscribe("/tf_distort/info", 1, &MocapFakeGPSPlugin::mocap_tf_params_cb, this);
+
+		mocap_tf_sub = mp_nh.subscribe("/vicon/DJI_450/DJI_450_drop", 1, &MocapFakeGPSPlugin::mocap_tf_cb, this);
+
+		
+	}
```

Comment:
Blank lines too. Why starts from `/`?
Two possibilities:

1. Use global node handle (i do not see any param readings) - e.g. `mp_nh()`
2. Use private nh with names like "out", "info" then use `<remap />`